tests/wake-format/basic/stdout

```diff
@@ -979,3 +975,16 @@ export def makeJSONRunner (plan: JSONRunnerPlan): Runner =
         )
 
     json
+
+def anyArray =
+    exists (_.getJArray | isSome) value
+
+def a =
+    rec Nil (if indent > 0 then Some "" else None) lhs
+
+def a = Some ""
+
+def a =
+    def x = if a then b, else !b
```

Comment:
I still maintain a dislike for single-line unary `,`, and this type error kind of emphasizes that.  Formatting looks good, though.

Reply:
Yeah, we can bikeshed the policy on unary `,` at some point but I'd like to push that to post MVP